iamproperty/iamproperty.github.io
# Testimonials component

## Summary of Changes:
1. Testimonials component
2. Unit tests/Visual tests

## Rationale
A group of testimonials collected into an interactive component

## How to Test
Open up the preview site (see the below netlify comment below) and navigate to /components/testimonial

## Ticket(s)
DS-56

## Checklist

## Notes
